CHANGELOG.md

```diff
@@ -7,13 +7,52 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [2.7.0] - November, 22, 2022
+## [2.8.0] - November, 30, 2022
 
 ### Design kit
 
 #### Added
 
 - [Icons] New icon UI/Navigation/swap-user
+
+### React Components
+
+#### Changed
+
+- [CookieConsent] Text changes for the Finnish and Swedish language versions
+
+#### Fixed
+
+- [Dropdown] Scrollbar becomes visible also with five options and the UI won't break
+- [DateInput] No more errors when minDate is over ten years in the future
+- [CookieConsent] No more errors from React.Portal if cookie consent modal is re-initialized or re-rendered after it has been closed.
+- [TimeInput] No more console warning about time format
+
+### Core
```

Comment:
I think the whole Core section can be removed? All of these component changes are only in hds-react.